src/Altinn.Platform/Altinn.Platform.Events/Events/appsettings.json

```diff
@@ -14,6 +14,7 @@
   },
   "PlatformSettings": {
     "ApiRegisterEndpoint": "http://localhost:5101/register/api/v1/",
+    "ApiProfileEndpoint": "http://localhost:5101/profile/api/v1/",
```

Comment:
values.yaml in altinn-studio-ops must also be updated based on his change.

Reply:
Done